src/main/java/com/ericsson/eiffel/remrem/protocol/MsgService.java

```diff
@@ -0,0 +1,45 @@
+package com.ericsson.eiffel.remrem.protocol;
+
+import com.google.gson.JsonObject;
+
+public interface MsgService {
+
+    /**
+     * Generates the event as string from the input json Object based on the
+     * message type.
+     * 
+     * @param StringmsgType
+     * @param JsonObject bodyJson
+     * @return the generated event as string format
+     */
+    String generateMsg(String msgType, JsonObject bodyJson);
+
+    /**
+     * Returns the Event Id from the json object.

```

Comment:
json object doesn't say anything. bodyJson is also a non descriptive term. Shouldn't we use jsonMessage and explain this is a message as defined in the used library in json format.